docs/models.yml

```diff
@@ -1623,16 +1565,9 @@ option:
   content_object_id:
     type: generic-relation
     to:
-      collections:  # Now, we have multiple models to vote about.
-        - collection: motion
-          field: option_ids
-        - collection: topic
-          field: option_ids
-        - collection: user
-          field:
-            name: option_$_ids
-            type: structured-relation
-            replacement: meeting_id
+      - motion/option_ids
+      - topic/option_ids
+      - user/option_$_ids
```

Comment:
Replacement is missing. Here it is impossible to guess that meeting_id is meant.

Reply:
The same for many other places ...

Reply:
The replacement is defined in the field `user/option_$_ids` as `meeting_id`